frc_characterization/newproject/__init__.py

```diff
@@ -297,7 +332,14 @@ def isRotation(units):
     teamNumberEntry.grid(row=2, column=1, sticky="ew")
 
     Label(topFrame, text="Unit Type:", anchor="e").grid(row=2, column=2, sticky="ew")
-    unitChoices = {"Feet", "Inches", "Meters", "Radians", "Rotations"}
+    unitChoices = [
+        Units.DEGREES.value,
+        Units.ROTATIONS.value,
+        Units.RADIANS.value,
+        Units.FEET.value,
+        Units.INCHES.value,
+        Units.METERS.value,
+    ]
     unitMenu = OptionMenu(topFrame, STATE.units, *sorted(unitChoices))
```

Comment:
as before, use `*sorted(Units.__members__.keys())`

[you could also do `*sorted(unit.value for unit in Units)` if that's more clear]

Reply:
[Done](https://github.com/Piphi5/frc-characterization/blob/Refactor/frc_characterization/newproject/__init__.py#L333)